packages/plugin-commands-outdated/test/index.ts

```diff
@@ -190,6 +190,65 @@ https://github.com/kevva/is-positive#readme
   }
 })
 
+test('pnpm outdated: format json', async () => {
+  tempDir()
+
+  await fs.mkdir(path.resolve('node_modules/.pnpm'), { recursive: true })
+  await fs.copyFile(path.join(hasOutdatedDepsFixture, 'node_modules/.pnpm/lock.yaml'), path.resolve('node_modules/.pnpm/lock.yaml'))
+  await fs.copyFile(path.join(hasOutdatedDepsFixture, 'package.json'), path.resolve('package.json'))
+
+  {
+    const { output, exitCode } = await outdated.handler({
+      ...OUTDATED_OPTIONS,
+      dir: process.cwd(),
+      format: 'json',
+    })
+
+    expect(exitCode).toBe(1)
+    expect(stripAnsi(output)).toBe(`{
+\t"@pnpm.e2e/deprecated": {
+\t\t"current": "1.0.0",
+\t\t"latest": "Deprecated"
+\t},
+\t"is-negative": {
+\t\t"current": "1.0.0",
+\t\t"latest": "2.1.0"
+\t},
+\t"is-positive": {
+\t\t"current": "1.0.0",
+\t\t"latest": "3.1.0",
+\t\t"dependencyKind": "dev"
+\t}
+}`)
+  }
+
+  {
+    const { output, exitCode } = await outdated.handler({
+      ...OUTDATED_OPTIONS,
+      dir: process.cwd(),
+      long: true,
+      table: false,
+    })
+
+    expect(exitCode).toBe(1)
+    expect(stripAnsi(output)).toBe(`@pnpm.e2e/deprecated
+1.0.0 => Deprecated
+This package is deprecated. Lorem ipsum
+dolor sit amet, consectetur adipiscing
+elit.
+https://foo.bar/qar
+
+is-negative
+1.0.0 => 2.1.0
+https://github.com/kevva/is-negative#readme
+
+is-positive (dev)
+1.0.0 => 3.1.0
+https://github.com/kevva/is-positive#readme
+`)
```

Comment:
why is this needed? This format already existed before your changes. Why do you add a test for it?

Reply:
Sorry, just mistake. [c92fd62](https://github.com/pnpm/pnpm/pull/5582/commits/c92fd62d66ceb9097e2311e0e80c4f3499fec38c) :+1: